cmd/gossamer/flags.go

```diff
@@ -285,6 +285,13 @@ var (
 		Usage: "Retain number of block from latest block while pruning",
 		Value: 256,
 	}
+
+	// GCModeFlag sets the blockchain GC mode. It's either full or archive.
+	GCModeFlag = cli.StringFlag{
+		Name:  "gcmode",
```

Comment:
can you rename this to `--pruning` to match substrate?

Reply:
Done